README.md

```diff
@@ -14,7 +14,13 @@
 ## Example usage
 
 ```
-./celestia-server --enable-rpc --rpc-addr $RPC_ADDR --rpc-port $RPC_PORT --celestia.auth-token $AUTH_TOKEN --celestia.gas-price $GAS_PRICE --celestia.gas-multiplier $GAS_MULTIPLIER --celestia.namespace-id $NAMESPACEID --celestia.rpc $CELESTIA_NODE_ENDPOINT
+./celestia-server --enable-rpc --rpc-addr $RPC_ADDR \
```

Comment:
This is just to improve UX of the readme and avoid people sidescrolling, so they can see things at once